src/pages/user/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from "@apollo/client";
+import React from "react";
+import { BackButton } from "../../components/back-button";
+import { DetailsCardShimmer } from "../../components/details-card-loading";
+import { SectionHeader } from "../../components/section-header";
+import { UserCard } from "../../components/user-card";
+import { userQuery } from "../../data/graphql/queries/user";
+import "./style.css";
+
+export const User: React.FC = () => {
```

Comment:
```suggestion
export const UserDetailPage: React.FC = () => {
```
quando vc só deixa `User`, por convenção, parece que é uma interface do domínio e não um componente. por isso é legal colocar um `View` ou `Page` 👍

Reply:
Valeu Beto, corrigido